### ISSUE TITLE
v14 - Notes

### ISSUE THREAD

**Issue by DocCyblade** [DocCyblade/tkl-odoo#50]
While I think this is the best release so far, it could have bug (probably) and need to iron out some other things like config tweaking, documentation.

Use this issue to make notes on the above

**Comment by JedMeister** [comment]
Awesome work! :+1:

**Comment by DocCyblade** [comment]
@JedMeister - When you have a moment see #59  and #49

**Comment by DocCyblade** [comment]
@JedMeister Testing is going well. Any [bugs are being marked](https://github.com/DocCyblade/tkl-odoo/issues?utf8=✓&q=is%3Aopen+milestone%3Av14-READY+label%3Abug+) we have a few hopefully we can smash them.

**Comment by DocCyblade** [comment]
@l-arnold - I am done for the night and will try to look at the other "bugs" in the early AM

**Comment by l-arnold** [comment]
I am almost done as well.

Everything seems to be working pretty well for me.  Doing LVM install now w/ 1 gb Ram, 1 Processor and not changes to openerp-server.conf file now.

**Comment by DocCyblade** [comment]
@l-arnold - I an very please at how things are progressing. I just need @JedMeister to give his two cents on some items, and I'll more than likely update the config file with some changes and more documentation regarding tweaking. after that, I'll do a test build and we mark this as v14-READY Once that is done, have to wait and see when Odoo will launch from this repo to official status.

Also, Landis you are a testing beast master! It's been a pleasure to work with you on this!

**Comment by l-arnold** [comment]
As well Ken, truly a pleasure.  This has been the bomb in terms of learning things the last days.

Lots too this but I think it is going to rock.

Thanks for your very hard and detailed work on everything.  I need to study your methods.  Need to particularly understand "mile stones" and related "generational markers".

(point being that rc1 is a transient thing on the way to the next.  Need to figure how to "get there" and see what is in front and what is behind.

Thanks so much.  Will turn in for tonight after a few comments on other issues.

**Comment by DocCyblade** [comment]
@JedMeister - I think the testing is pretty much done. I am making a few changes and documentation inside the config file. Once thing I changed was the workers = 0 I did this because we don't really know what hardware it will be on and this just threaded mode. We need to test this

I will push these to dev-14-ready branch for final testing

@l-arnold - One last testing! I open #61 and assigned it to you. Just follow the test check list. Check off each one and just put the numbers inline where the `xxMB` for real and swap memory. 

Depending on the result from #61 I would love to make this v14-READY today if we can get this tested

**Comment by DocCyblade** [comment]
I'll check back in later. Off to do some "real work" :smile:

**Comment by l-arnold** [comment]
Making now on dev-14-ready

Doing some other projects here today as well so will be coming and going.  For sure finished by Sunday AM but will push today as well.

**Comment by DocCyblade** [comment]
Thanks Landis !

**Comment by l-arnold** [comment]
Still working on the Roof.  Will work late and likely very early tommorrow on this.
Sorry for the delay.

Got the Build done and downloaded.

**Comment by DocCyblade** [comment]
No worries :-) 

I am running the same test as well ill post mine in another issue

**Comment by DocCyblade** [comment]
#62 is done. Seems to work pretty well on workers = 0 (in threaded mode instead of process mode) Also this test seems to point out that 1GB of ram would be ideal, as 512 is just not enough. See the usage in #62 I don't think it's taking full advantage of the two cores but the workers option set to 0 make it more flexible in installing it in TKL. 

@JedMeister - You think Alon would go for a 1GB VM? 512 works but it's eating into swap right from the start. It seems 650MB is where it floats at, and can go up from there with usage.

**Comment by DocCyblade** [comment]
@l-arnold - Not sure if you started testing but, I am pretty confident about this release. I want to know what you think, and if you give the :+1:  for this release.

**Comment by DocCyblade** [comment]
I'll be back in the Sunday, AM if you finish #61  close it and tell me what you think :+1:  or :-1:  here in this issue. If things look good to you I'll move this on to dev then onto master as v14-READY!

**Comment by l-arnold** [comment]
Strange,  can't build htop today.  Could yesterday.

**Comment by DocCyblade** [comment]
@l-arnold you doing 

```
apt-get update
apt-get install htop
```

**Comment by l-arnold** [comment]
I had not run the update part or updated the packages at install.

That explains it.

**Comment by JedMeister** [comment]
Yep we can give it 1GB I reckon...

**Comment by DocCyblade** [comment]
@JedMeister and @l-arnold - Take a look at #63 and #64  let me know your thoughts on should we go with what we got (I think it pretty good) or wait until both or one or the other. Thanks. Depending on your thoughts I may wait and implement one or both. I'll check back in the evening my time see what your thoughts are. I would like an unanimous decision on both or one or the other, as I would like for all of us to be on board.

**Comment by l-arnold** [comment]
About Half Done.  Building all the VM's just now to finish.

I didn't see a note on the "workers = " question.  Right now it appears there is both

```
           workers=2
```

and 
               workers=0 

in openerp-server.conf file.
Assuming I should be testing this on

```
         workers=0
```

So we can see "min" case.

**Comment by DocCyblade** [comment]
@l-arnold @JedMeister PR #66 has been open, once Landis merges, I'll tag it v14-READY and you and the core devs can use that tagged release as the base for Odoo being considered. 

Closing this issue
